deno/fhirchat-demo/hooks/aiConversationContext.ts

```diff
@@ -85,12 +89,27 @@ function createAIConversationContext(): AIConversationContext {
     } else {
       console.log("Websocket not connected.");
     }
+    setAgentMockResponse();
```

Comment:
While this mocking works, it does not replace the actual service, making the actual _real_ service actually try to make the real thing in parallel, which is not ideal...

So, there would be an extra effort needed here to make the app switchable between only mock or only real.  Also, proper comments would be needed in code to make it clear that some scaffolds are in place.

But, I the initial proposition was to have the mock operate _in-place_ of the AI component in the backend, not requiring any state management change to the frontend app and likely making it more chirurgical & simple.  
At this point some efforts have already been applied to mocking on the frontend, but since the _state_ management is not clean anyway and would require more work to be complete, I believe moving the mocking to the backend code in `routes/api/aiConversation.ts:35` would be simple & easier to switch & get away from when we are done.

I can assist with the move if it sounds like an headache 😄